src/Amplitude.js

```diff
@@ -254,6 +258,26 @@
             return false;
         }
 
+        function convertJsonAttrs(customAttributes) {
+            if (forwarderSettings.sendEventAttributesAsObjects === 'True') {
+                for (var key in customAttributes) {
+                    if (typeof customAttributes[key] === 'string') {
+                        try {
+                            var parsed = JSON.parse(customAttributes[key]);
+                            if (typeof parsed === 'object') {
```

Comment:
Just checking - this works for arrays too? Looks like you're testing for arrays in the unit tests but just want it be sure

Reply:
Yep. In JS When you do typeof [1, 2, 3], it returns 'object' because an array is an object. To test for arrays specifically, you'd do Array.isArray([]) //true. but Since we are testing for both objects and arrays, that's not necessary here since we're not distinguishing between the two of them for other logic purposes, we're just setting the parsed object/array to the value of the key.